index.js

```diff
@@ -6,43 +6,91 @@ var app = new Alexa.app('carvis');
 var FAADataHelper = require('./faa_data_helper');
 var rideHelper = require('./ride-helper');
 
-// we would have the same -- res.say(prompt) -- "do you want the fastest or cheapest ride" || "which type of ride (pool, X, select ... )"
-// no endsession, as utterances (destination)
 app.launch(function(req, res) {
-  var prompt = 'For delay information, tell me an Airport code.';
+  // NOTE: we may have to split up our query into multiple utterances --
+  // cheapest || fastest
+  // type of car
+  // pickup (and validation)
+  // destination (and validation)
+  // --> all this would be much shorter if we use presets from a webapp.
+  // --> can also use user location from the phone, but less accurate.
+
+  var prompt = 'With CARVIS you can order the cheapest or fastest ride available. Tell me CHEAPEST or FASTEST and your start and end destinations';
+
```

Comment:
lets start with supporting just the simplest type of utterance? so instead, the prompt could say something like "With CARVIS you can order the cheapest or fastest ride available. For example, you can say 'carvis, find me the cheapest car to Hack Reactor'"